web/src/lib/components/shared-components/navigation-bar/navigation-bar.svelte

```diff
@@ -44,11 +44,8 @@
   <div
     class="grid h-full grid-cols-[theme(spacing.18)_auto] items-center border-b bg-immich-bg py-2 dark:border-b-immich-dark-gray dark:bg-immich-dark-bg md:grid-cols-[theme(spacing.64)_auto]"
   >
-    <a data-sveltekit-preload-data="hover" class="mx-4 flex place-items-center gap-2 md:mx-6" href={AppRoute.PHOTOS}>
-      <ImmichLogo height="35" width="35" />
-      <h1 class="hidden font-immich-title text-2xl text-immich-primary dark:text-immich-dark-primary md:block">
-        IMMICH
-      </h1>
+    <a data-sveltekit-preload-data="hover" class="ml-4" href={AppRoute.PHOTOS}>
```

Comment:
The logo text should be hidden on smaller screens like before, otherwise the logo becomes too small